pkg/authorize/authorizer.go

```diff
@@ -0,0 +1,55 @@
+package authorize
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/dgrijalva/jwt-go"
```

Comment:
Just to make sure i understood @trillnye's comments last thursday, the plan is to use JWTs for backend interactions but (for now at least) the frontend will work make calls to application-layer API without any authentication?

Reply:
That is correct -- we've decided to figure out that stuff later, but for the case of this API i'm going to continue with using `jwt`s and sharing a secret across applications